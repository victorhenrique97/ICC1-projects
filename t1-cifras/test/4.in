177
NO ENTANTO, NAO PODEMOS ESQUECER QUE A CONSULTA AOS DIVERSOS MILITANTES OBSTACULIZA A APRECIACAO DA IMPORTANCIA DO SISTEMA DE FORMACAO DE QUADROS QUE CORRESPONDE AS NECESSIDADES
1
17
