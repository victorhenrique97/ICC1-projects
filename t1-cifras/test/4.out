EF VEKREKF, ERF GFUVDFJ VJHLVTVI HLV R TFEJLCKR RFJ UZMVIJFJ DZCZKREKVJ FSJKRTLCZQR R RGIVTZRTRF UR ZDGFIKRETZR UF JZJKVDR UV WFIDRTRF UV HLRUIFJ HLV TFIIVJGFEUV RJ EVTVJJZURUVJ
